pyramid/scripts/pcreate.py

```diff
@@ -154,5 +169,48 @@ def out(self, msg): # pragma: no cover
         if not self.quiet:
             print(msg)
 
+    def validate_input(self):
+        if not self.options.scaffold_name:
+            self.out('You must provide at least one scaffold name: -s <scaffold name>')
+            self.out('')
+            self.show_scaffolds()
+            return False
+        if not self.args:
+            self.out('You must provide a project name')
+            return False
+        available = [x.name for x in self.scaffolds]
+        diff = set(self.options.scaffold_name).difference(available)
+        if diff:
+            self.out('Unavailable scaffolds: %s' % ", ".join(list(diff)))
+            return False
+
+        pkg_name = self.project_vars['package']
+
+        if pkg_name == 'site' and not self.options.force_bad_name:
+            self.out('The package name "site" has a special meaning in '
+                     'Python. Are you sure you want to use it as your '
+                     'project\'s name?')
+            return self.confirm_bad_name('Really use "{0}"?: '.format(pkg_name))
```

Comment:
Yeah, but site isn't a package, so it deserves a special treatment IMO, otherwise rookie devs will try to find a "site" package in their current env and wonder what the general error message ('Package "site" already exists...') is supposed to mean